sphinx-config/prog_models_guide.rst

```diff
@@ -60,6 +60,22 @@ In a ProgPy model, this state transition can be represented one of two ways, eit
     :width: 70 %
     :align: center
 
+States can also be discrete or continuous. :term:`Discrete states<discrete state>` are those which can only exist in a finite set of values. Where continuous states are those initialized with a number, discrete states are initialized using the function :py:func:`progpy.create_discrete_state`, like the examples below. Each discrete state represents a unique condition or mode, and transitions between states are governed by defined rules or events, providing clarity and predictability in state management.
+
+.. code-block:: python
+
+    >>> from progpy import create_discrete_state
+    >>> ValveState = create_discrete_state(2, ["open", "closed"])
+    >>> x["valve"] = ValveState.open
+
+.. code-block:: python
+
+    >>> from progpy import create_discrete_state
+    >>> GearState = create_discrete_state(5, transition="sequential")
+    >>> x["gear"] = GearState(1)
+
+.. note::
+    :term:`Discrete states <discrete state>` are different from :term:`discrete models <discrete model>`. Discrete models are models where state transition is discrete, where discrete states are where the state itself is discrete. Discrete models may have continuous states.
```

Comment:
Super small nit but any chance we could remove the where in the middle before discrete states? The multiple uses of where made me do a double take.